presto-hive/src/test/java/io/prestosql/plugin/hive/s3/TestPrestoS3FileSystem.java

```diff
@@ -155,7 +156,7 @@ public void testAssumeRoleCredentials()
         }
     }
 
-    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "S3 credentials not configured")
+    @Test
     public void testInstanceCredentialsDisabled()
```

Comment:
Please rename the test method to eg `testDefaultCredentials`.

Reply:
Will do